### DISCUSSION TITLE
Release Error after Upgrade to Deployer 7.0.2

### DISCUSSION THREAD

**Discussion by sebastian-ehrling** [deployphp/deployer#3298] [Help needed]
- Deployer version: 7.0.2
- Deployment OS: LINUX / UBUNTU 20

After Upgrade from Deployer 7.0.1 to 7.0.2 i get the following error:
task deploy:release
[prod_develop]  Deployer\Exception\Exception  in release.php on line 115:
[prod_develop]
[prod_develop]   Release name "2022-09-12_12-40-34" already exists.
[prod_develop]   Release name can be overridden via:
[prod_develop]   dep deploy -o release_name=...
[prod_develop]
task deploy:failed
task deploy:unlock

**Comment by antonmedv** [comment]
Please, check:

> Release name "2022-09-12_12-40-34" already exists.

**Reply by sebastian-ehrling** [reply]
Hi @antonmedv 
yes i've check this. The directory doesn't exit before Deployment.

**Reply by antonmedv** [reply]
¯\\\_(ツ)_/¯

**Comment by codekandis** [comment]
This issue has been mentioned in https://github.com/deployphp/deployer/issues/3301. Maybe my answer helps you. In fact - check for task redundancy while some tasks are just a couple of other tasks. Maybe your tasks double the release task.
